src/transformers/models/llava/processing_llava.py

```diff
@@ -125,8 +139,27 @@ def __call__(
               `None`).
             - **pixel_values** -- Pixel values to be fed to a model. Returned when `images` is not `None`.
         """
+        if images is None and text is None:
+            raise ValueError("You have to specify at least images or text.")
+        # check if images and text inputs are reversed for BC
+        if (
+            text is not None
+            and not isinstance(text[0], str)
```

Comment:
`text` could also just be an `str`

Reply:
The check should still work in that case, except if we have an empty string. I will try to think of something cleaner